pkg/scanners/clairify/clairify.go

```diff
@@ -44,12 +44,16 @@ const (
 var (
 	_ scannerTypes.Scanner                  = (*clairify)(nil)
 	_ scannerTypes.ImageVulnerabilityGetter = (*clairify)(nil)
-)
 
-var (
-	log = logging.LoggerForModule()
+	log             = logging.LoggerForModule()
+	scannerEndpoint = fmt.Sprintf("scanner.%s.svc", env.Namespace.Setting())
 )
 
+// GetScannerEndpoint returns the scanner endpoint with a configured namespace. env.ScannerGRPCEndpoint is only used by Sensor.
```

Comment:
`env.ScannerGRPCEndpoint` should probably also be `scanner.<namespace>.svc:8443`, right? or perhaps we should not use that at all and just use this instead. Otherwise, wouldn't OpenShifts using non-`stackrox` namespaces has "local scanning" issues?

Reply:
The default for the scanner endpoint is set by Helm in Sensor's deployment spec. The `ROX_NAMESPACE` env var which the function depends on is never set on Sensor, thus has no effect.

https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-secured-cluster/templates/sensor.yaml.htpl#L124-L129

We can align this behaviour but would prefer to do it in a different PR to merge this one first.